.cirrus.yml

```diff
@@ -229,6 +233,7 @@ task:
     cpu: 4
   env:
     CIRRUS_WORKING_DIR: "C:\\Windows\\Temp\\flutter sdk"
+    PATH: "$CIRRUS_WORKING_DIR/bin;$CIRRUS_WORKING_DIR/bin/cache/dart-sdk/bin;$PATH"
```

Comment:
Aren't Windows environment variables expressed as `%ENV_VAR%`?

Reply:
I copied and pasted this from elsewhere in the same file and it fixed my problem. I've no idea what the syntax of this string is supposed to be or how it's implemented.